src/devices_models/devices/DC_branches.jl

```diff
@@ -86,10 +86,7 @@ function branch_rate_constraints!(
     psi_container::PSIContainer,
     devices::IS.FlattenIteratorWrapper{B},
     model::DeviceModel{B, HVDCLossless},
-    system_formulation::Union{
-        Type{<:PM.AbstractActivePowerModel},
-        Type{<:PM.AbstractPowerModel},
-    },
+    system_formulation::Type{<:PM.AbstractPowerModel},
```

Comment:
@jd-lara Not sure if you had a reason for this.  The union shouldn't be necessary.

Reply:
This is not necessary, I think this was introduced by @claytonpbarrows in an earlier network refactor.